eland/ml/imported_ml_model.py

```diff
@@ -75,7 +75,6 @@ class ImportedMLModel(MLModel):
         - xgboost.XGBClassifier
             - only the following objectives are supported:
                 - "binary:logistic"
-                - "binary:hinge"
```

Comment:
Is there anything we can do to support "binary:hinge"? If so we could add an issue to implement support.

Reply:
I can try some more, but all the different variants failed periodically. One would think since leaf values are `0/1` that our typical prediction logic would work :(

Reply:
hinge won't work without some backend changes :(